postrender.go

```diff
@@ -73,6 +74,25 @@ func applyMarkdown(data []byte, baseURL string, cfg *config.Config) string {
 	return applyMarkdownSpecial(data, false, baseURL, cfg)
 }
 
+func disableYoutubeAutoplay(outHTML string) string {
+	for _, match := range youtubeReg.FindAllString(outHTML, -1) {
+		u, err := url.Parse(match)
+		if err != nil {
+			log.Error("Couldn't parse youtube url: %v", err)
```

Comment:
If there's an error, we should probably move on to the next match (by adding a `continue` statement here).

Reply:
Thanks for the comment! I Will update that.